src/server.c

```diff
@@ -5843,6 +5844,7 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "pubsubshard_channels:%llu\r\n", kvstoreSize(server.pubsubshard_channels),
                 "latest_fork_usec:%lld\r\n", server.stat_fork_time,
```

Comment:
Not specifically related to this PR, but this info field is confusing. We thought it was how long the fork latest, but apparently it just covers the system call time. This made the info name you proposed also uses fork.

Reply:
yes, it is indeed confusing at the first time. A field show `how long the fork latest` also will be helpful.